www/_data/toc/de-dev-manual.yml

```diff
@@ -25,12 +25,12 @@
                 -   name: "Windows"
                     url: "guide/platforms/win8/index.html"
                 -   name: "WP8"
-                    url: "guide/platforms/wp8/home.html"                                                           
+                    url: "guide/platforms/wp8/home.html"
         -   name: "Manage versions and platforms"
             url: "platform_plugin_versioning_ref/index.html"
         -   name: "Debug and test"
             url:
-        -   name: "Use icons and splash screens"
```

Comment:
The guideline is to keep the first letter in caps only for the first word and to have the remaining words begin in smallcase. Can you please also change it for other occurrences in the TOC?